service/src/main/resources/config/verily/sdd/expanded/entity/person.json

```diff
@@ -94,29 +94,54 @@
           },
           "display" : "Hispanic or Latino"
         },
-        "count" : 113110
+        "count" : 113133
       }, {
         "enumVal" : {
           "value" : {
             "int64Val" : 0
           },
           "display" : "No matching concept"
         },
-        "count" : 960195
+        "count" : 960199
       }, {
         "enumVal" : {
           "value" : {
             "int64Val" : 38003564
           },
           "display" : "Not Hispanic or Latino"
         },
-        "count" : 2375593
```

Comment:
It seems surprising that these counts changed when you were just adding some new columns. Any idea why?

Reply:
Good catch! It turns out `x_biovu_sample_status` had duplicate rows for some people. For example, person 4587323 had 11 duplicate rows. (I spot-checked a couple other people; rows were duplicates.)

I changed `person.sql` to take out duplicate rows from `x_biovu_sample_status`. These count changes reverted.